modules/library/main/src/test/java/org/geotools/filter/FiltersTest.java

```diff
@@ -191,7 +192,12 @@ public void testFindPropertyNameEmpty() {
 
     @Test
     public void testEmptyEscape() {
-        PropertyIsLike like = ff.like(ff.literal("abc def"), "*de*", "*", "_", "");
-        assertTrue(like.evaluate(null));
+        try {
+            PropertyIsLike like = ff.like(ff.literal("abc def"), "*de*", "*", "_", "");
+            assertTrue(like.evaluate(null));
+            fail();
```

Comment:
This logic is a bit confusing. Perhaps remove the `assertTrue`?

Reply:
that can go now, I thought I had a problem with it not being evaluated, but turns out the bug isn't there in Java 11